LEAF_Request_Portal/sources/Form.php

```diff
@@ -427,33 +408,36 @@ public function newForm($userID)
     public function getIndicator($indicatorID, $series, $recordID = null, $parseTemplate = true)
```

Comment:
Worked on returns and hinting, there were some that I needed to remove hinting and remove return typing. I think this should suffice for a first pass since some of these functions will need some rework.